### PR TITLE
fix 1248: parse json theme string for standalone tag

### PR DESCRIPTION
**Pull request by langhabel**
Hi,
this PR fixes the problem described in #1248, by parsing the theme string as json.

Example:
```
<html>

<body>
  <redoc spec-url="../demo/openapi.yaml" native-scrollbars theme='{"colors": {"primary": { "main": "#FF5733"}}}'></redoc>
  <script src="../bundles/redoc.standalone.js"></script>
</body>

</html>
```

### CODE REVIEW

**Comment by Shahaed** [comment]
Please review and merge

File: src/standalone.tsx [hunk 1]
```diff
@@ -32,7 +32,13 @@ function parseOptionsFromElement(element: Element) {
...
   for (const attrName in attrMap) {
     const optionName = attrName.replace(/-(.)/g, (_, $1) => $1.toUpperCase());
-    res[optionName] = attrMap[attrName];
+    let attr = attrMap[attrName];
+
+    if (attrName === 'theme') {
+      attr = JSON.parse(attr);
+    }
+
+    res[optionName] = attr;
```
Comment (RomanHotsiy) [inline src/standalone.tsx:41]:
```suggestion
    const attr = attrMap[attrName];
    res[optionName] = attrName === 'theme' ? JSON.parse(attr) : attr;
```

File: src/standalone.tsx [hunk 2]
```diff
@@ -32,7 +32,8 @@ function parseOptionsFromElement(element: Element) {
   const res = {};
   for (const attrName in attrMap) {
     const optionName = attrName.replace(/-(.)/g, (_, $1) => $1.toUpperCase());
-    res[optionName] = attrMap[attrName];
+    const attr = attrMap[attrName];
+    res[optionName] = attrName === 'theme' ? JSON.parse(attr) : attr;
```
Comment (RomanHotsiy) [inline src/standalone.tsx:36]:
```suggestion
    const optionValue = attrMap[attrName];
    res[optionName] = attrName === 'theme' ? JSON.parse(optionValue) : optionValue;
```

**Comment by RomanHotsiy** [comment]
The new version will be published later this week. Thanks for the PR! and thanks for bumping @Shahaed !

**Comment by godd9170** [comment]
> The new version will be published later this week. Thanks for the PR! and thanks for bumping @Shahaed !

Did this ever end up getting published @RomanHotsiy?

**Comment by mrrhysbutler** [comment]
Yeah it would be great if we could cut a new version with this fix. 🙏

**Comment by StarfallProjects** [comment]
Is this still unpublished? I just grabbed the latest version and it doesn't seem to be working, but could also be user (me) error. Would be good to know before I spend too much time on it.

**Comment by studouglas-scx** [comment]
I can confirm this is working in 2.0.0-rc.56, but not in @next (as of Sept 24)

**Pull request merged**